travvy/src/components/Home.vue

```diff
@@ -24,9 +24,9 @@
     <!-- data binding dates, corresponding data object below -->
     <label for="Dates">     Select Departure Date:</label>
     <!-- user selects dates they are travelling in --> 
-      <input type="text" class="dates"> 
+      <input type="Date" class="dates"> 
```

Comment:
So great that you were able to convert these input boxes to be treated as date selectors!